src/Geno/C++/GUI/Widgets/WorkspaceOutliner.h

```diff
@@ -49,4 +51,7 @@ class WorkspaceOutliner
 
 	bool                  m_ExpandWorkspaceNode     = false;
 
+	CommandStack m_UndoCommandStack = {};
+	CommandStack m_RedoCommandStack = {};
```

Comment:
Why do we use two different command stacks for undo and redo?

Reply:
When we do undo we add a redo command so we cant add that redo command in undocommandstack as it will already have another commands for undo so we will use another stack and during redo we add a undo command, therefore we need to use two stacks

Reply:
Ah, I see. That approach works for now, but I think I want to revisit that in the future so that we only need one stack.